docs/reverse_proxy.md

```diff
@@ -146,3 +158,10 @@ connecting to Synapse from a client.
 Synapse exposes a health check endpoint for use by reverse proxies.
 Each configured HTTP listener has a `/health` endpoint which always returns
 200 OK (and doesn't get logged).
+
+## Synapse administration endpoints
+
+Endpoints for administering your Synapse instance are placed under
+`/_synapse/admin`. These require authentication through an access token of an
+admin user. Nevertheless, it is not advised to expose these ports to the public
+internet.
```

Comment:
I'd phrase this less strongly. 

"As access to these endpoints grants the caller a lot of power, we do not recommend exposing them to the public internet without good reason." 

or words to that effect